### PR TITLE
Card installation

### PR DESCRIPTION
**Pull request by archit7448**

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
The code looks good to me. Please add a description to the PR's.  Code looks good to merge.

File: Components/Badges/badges.html [hunk 1]
```diff
@@ -72,6 +72,7 @@
         </nav>
         <div class="aside-wrapper">
           <h1 class="heading">BADGES</h1>
+          <h2 class="text-grey">Different type of badges used for notification</h2>
           <div class="flex-col copy">
```
Comment (iRohitGaur) [inline Components/Badges/badges.html:76]: can you explain the "copy" class?

**Review by iRohitGaur — APPROVED** [review]
Good to merge

File: Components/Badges/badges.html (hunk 1 above)
Comment (archit7448) [reply Components/Badges/badges.html:76]: copy class is to copy the code snippet Javascript is working

**Pull request merged**